queries.sql

```diff
@@ -6,5 +6,35 @@ SELECT name FROM Animals WHERE neutered = TRUE AND escape_attempts < 3;
 SELECT date_of_birth FROM Animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM Animals WHERE weight_kg > 10.5;
 SELECT * FROM Animals WHERE neutered = TRUE;
-SELECT * FROM Animals WHERE name<> 'Gabumon';
-SELECT * FROM Animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM Animals WHERE name <> 'Gabumon';
+SELECT * FROM Animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+BEGIN;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
```

Comment:
* Some required transactions are still missing below are the missing transactions 👇 
```
Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Verify that changes were made.
Commit the transaction.
Verify that changes persist after commit.
Take a screenshot of the results of your actions.
Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
Take a screenshot of the results of your actions.
```
* This is an example transaction query below 👇 UPDATE the animals table by setting the species column to unspecified, then roll back the change (**for this step you are explicitly asked to verify that change was made and rollbacked, so make sure that they included all steps as in the example below **). **Query** :     
```
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone

```
* Kindly use the Hint provided above to write the missing queries